gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/MySqlDagManagementStateStoreTest.java

```diff
@@ -92,6 +95,16 @@ public static <T> boolean compareLists(List<T> list1, List<T> list2) {
     return true;
   }
 
+  @Test
+  public void testExistsLeasableEntity() throws Exception{
+    Mockito.when(leaseArbiter.existsLeasableEntity(Mockito.any(DagActionStore.LeaseParams.class))).thenReturn(true);
+    String flowName = "testFlow";
+    String flowGroup = "testGroup";
+    DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowName, flowGroup, System.currentTimeMillis(), "testJob", DagActionStore.DagActionType.LAUNCH);
+    DagActionStore.LeaseParams leaseParams = new DagActionStore.LeaseParams(dagAction);
+    leaseArbiter.existsLeasableEntity(leaseParams);
```

Comment:
No assertions? Also in  this unit test we are mocking leaseArbiter, so I don't think it adds any value on testing the functionality of leaseArbiter

Reply:
Thanks for pointing out missed adding the Assertion for this test, purpose of this test is to verify that MySQLStatemanagementStore returns the value returned by leaseArbiter, also updated this test.